CMakeLists.txt

```diff
@@ -401,6 +401,10 @@ set(LIBRARY_OUTPUT_PATH "bin")
 
 target_link_libraries(xmr-stak ${LIBS} xmr-stak-c xmr-stak-backend)
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+    set_property(TARGET xmr-stak PROPERTY LINK_FLAGS "/level='requireAdministrator' /uiAccess='false'")
```

Comment:
Does this mean that the miner must run always started as administrator under windows?